lnwallet/channel.go

```diff
@@ -2107,6 +2107,7 @@ func (lc *LightningChannel) ReceiveHTLCSettle(preimage [32]byte, logIndex uint64
 		Amount:      htlc.Amount,
 		RPreimage:   preimage,
 		ParentIndex: htlc.Index,
+		RHash:       htlc.RHash,
```

Comment:
The field have been already present in payment descriptor, I have added its initialisation. In the context of common sense if the field is present inside the structure programmer expects this filed to be initialised.